developer/src/kmcmplib/src/kmx_u16.cpp

```diff
@@ -96,7 +101,11 @@ std::string toHex(int num1) {
 			s += (87 + temp);
 		num = num / 16;
 	}
+#ifdef _MSC_VER
 	reverse(s.begin(), s.end());
+#else
+	std::reverse(s.begin(), s.end());
```

Comment:
I wonder if `std::reverse` would also work on Windows.

Reply:
It does seem to work ... I had adopted the more conservative change, but I'll switch to std::reverse for all.